From 6981ec201786d489b41dae0bb5c329cd90f2569e Mon Sep 17 00:00:00 2001
From: Olivier Leveque <olivier.leveque@intel.com>
Date: Wed, 18 Mar 2015 17:20:55 +0100
Subject: [PATCH] Add kernel log when watchdog daemon kicks TCO wdg

This adds in kernel ring buffer a log when watchdog daemon refreshes the TCO
watchdog. This is useful to debug TCO watchdog expiration issues leading to a
platform reset.

Change-Id: Ie2602cfbbddc95267425ea2fb927ab66c41dae7b
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1325
Signed-off-by: Olivier Leveque <olivier.leveque@intel.com>
---
 drivers/watchdog/iTCO_wdt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/watchdog/iTCO_wdt.c b/drivers/watchdog/iTCO_wdt.c
index e802a5428410..6f609f438769 100644
--- a/drivers/watchdog/iTCO_wdt.c
+++ b/drivers/watchdog/iTCO_wdt.c
@@ -266,6 +266,7 @@ static int iTCO_wdt_stop(struct watchdog_device *wd_dev)
 
 static int iTCO_wdt_ping(struct watchdog_device *wd_dev)
 {
+	pr_notice("%s\n", __func__);
 	spin_lock(&iTCO_wdt_private.io_lock);
 
 	iTCO_vendor_pre_keepalive(iTCO_wdt_private.smi_res, wd_dev->timeout);
-- 
2.1.0

